sdk/rust/oak/src/grpc/mod.rs

```diff
@@ -166,22 +166,29 @@ pub fn event_loop<T: OakNode>(
     let mut space = crate::new_handle_space(&read_handles);
 
     loop {
-        // Block until there is a message to read on an input channel.
+        // Block until there is a method notification message to read on an
+        // input channel.
         crate::prep_handle_space(&mut space);
         // TODO: Use higher-level wait function from SDK instead of the ABI one.
         let status =
             unsafe { oak_abi::wait_on_channels(space.as_mut_ptr(), read_handles.len() as u32) };
         crate::result_from_status(status as i32, ())?;
 
         let mut buf = Vec::<u8>::with_capacity(1024);
-        let mut handles = Vec::<Handle>::with_capacity(1);
+        let mut handles = Vec::<Handle>::with_capacity(2);
         crate::channel_read(grpc_in_handle, &mut buf, &mut handles)?;
-        if buf.is_empty() {
-            panic!("no bytes received")
+        if !buf.is_empty() {
+            error!("unexpected data received in gRPC notification message")
         }
-        if handles.is_empty() {
-            panic!("no response handle received alongside gRPC request")
+        if handles.len() < 2 {
```

Comment:
There should be __exactly__ two handles here, right? I think it would be better to error out if we received more.